plugin/api.go

```diff
@@ -264,6 +264,11 @@ type API interface {
 	// Minimum server version: 5.6
 	GetProfileImage(userId string) ([]byte, *model.AppError)
 
+	// SetProfileImage set user's profile image.
+	//
+	// Minimum server version: 5.6
+	SetProfileImage(userId string, data []byte) (bool, *model.AppError)
```

Comment:
What is the purpose of the returned `bool` here? If the plugin wants to check if the method succeeded, it can just check if the returned error is not `nil`.

Reply:
no reason 😂 will remove

Reply:
done